File: mobile/lib/domain/entities/recommendation.dart
enum RecommendationPriority {
  high,
  medium,
  normal,
}

class Recommendation {
  final String id;
  final String courseId;
  final String title;
  final String subtitle;
  final String targetSkillId;
  final String targetSkillName;
  final String targetNodeId;
  final String reason;
  final int suggestedMinutes;
  final RecommendationPriority priority;

  const Recommendation({
    required this.id,
    required this.courseId,
    required this.title,
    required this.subtitle,
    required this.targetSkillId,
    required this.targetSkillName,
    required this.targetNodeId,
    required this.reason,
    required this.suggestedMinutes,
    required this.priority,
  });
}

File: src/domain/entities/Recommendation.ts
export interface Recommendation {
  id: string;
  courseId: string;
  title: string;
  subtitle: string;
  targetSkillId: string;
  targetSkillName: string;
  targetNodeId: string;
  reason: string;
  suggestedMinutes: number;
  priority: 'high' | 'medium' | 'normal';
}
